### PR TITLE
Help Window UI update

### PR DESCRIPTION
**Pull request by ErvinK123**
Provides a brief summary of supported commands when opened. 
Changed link to bring straight to user guide, previously redirected to project website. 
Styling of components.
helpwindow

### CODE REVIEW

File: src/main/java/gim/ui/HelpWindow.java [hunk 1]
```diff
@@ -15,9 +16,23 @@
...
-    public static final String USERGUIDE_URL = "https://ay2223s1-cs2103t-t15-4.github.io/tp/";
-    public static final String HELP_MESSAGE = "Refer to the user guide: " + USERGUIDE_URL;
+    public static final String USERGUIDE_URL = "https://ay2223s1-cs2103t-t15-4.github.io/tp/UserGuide.html";
+    public static final String HELP_MESSAGE = USERGUIDE_URL;
 
+    public static final String HELP_PARA = "Hi there, welcome to GIM! Here is a list of supported commands.\n"
+            + "1) :a - Adds an exercise\n"
+            + "2) :d - Deletes an exercise\n"
+            + "3) :ls - Lists all exercise entries\n"
+            + "4) :find - Finds an exercise with a particular name\n"
```
Comment (24Donovan24) [inline src/main/java/gim/ui/HelpWindow.java:26]: :find do allow users to key in more than one name, perhaps we should include it in the description?

File: src/main/java/gim/ui/HelpWindow.java [hunk 2]
```diff
@@ -15,9 +16,23 @@
...
+            + "5) :clear - Clears the saved exercises and resets the data in the system\n"
+            + "6) :sort - Sorts the exercises by date, if they have the same date, by name \n"
+            + "7) :range - Displays the exercises saved over a range of days\n"
+            + "8) :pr - Displays the Personal Record associated with a certain exercise\n"
```
Comment (24Donovan24) [inline src/main/java/gim/ui/HelpWindow.java:30]: Similar thought as the above comment. :pr allows user to display all PRs?

File: src/main/java/gim/ui/HelpWindow.java (hunk 1 above)
Comment (ErvinK123) [reply src/main/java/gim/ui/HelpWindow.java]: Necessary changes have been made, thanks

File: src/main/java/gim/ui/HelpWindow.java (hunk 2 above)
Comment (ErvinK123) [reply src/main/java/gim/ui/HelpWindow.java]: Necessary changes have been made, thanks

**Pull request merged**